src/me/components/DocSearchWidget.scss

```diff
@@ -0,0 +1,178 @@
+$fontcolor: #666666;
+$black: #000;
+$sidebar-search-bg: rgb(70, 70, 82);
+$sidebar-search-shadow: rgba(#0f0e15, 0.5);
+$article-heading: #ffffff;
+$article-hr: #383846;
+$article-text: #d4d7dd;
+$article-link: #00a3ff;
+
+@import '@influxdata/clockface/dist/variables.scss';
+
+.WidgetSearch {
+  .WidgetHelperText {
+    text-align: right;
+  }
+  .DocSearch-Modal {
+    background-color: $g5-pepper !important;
+    box-shadow: none;
+    width: 100% !important;
+    min-width: 100% !important;
+    margin: 0 auto !important;
```

Comment:
nit: BEM classes & nix the `!important`?

Reply:
These need to be important to override the styles sent in from Algollia